src/foam/u2/dao/MDBrowserListView.js

```diff
@@ -0,0 +1,332 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.dao',
+  name: 'MDBrowserListView',
+  extends: 'foam.u2.View',
+  requires: [
+    'foam.comics.SearchMode',
+    'foam.comics.v2.DAOControllerConfig',
+    'foam.log.LogLevel',
+    'foam.u2.ActionView',
+    'foam.u2.dialog.Popup',
+    'foam.u2.layout.Cols',
+    'foam.u2.layout.Rows',
+    'foam.u2.layout.MDSearchView',
+    'foam.u2.view.ScrollTableView',
+    'foam.u2.view.SimpleSearch',
+    'foam.u2.view.TabChoiceView',
+    'foam.u2.layout.MDToolbarView',
+    'foam.u2.ToolbarAction'
+  ],
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  documentation: `
+    List controller with toolbar on top.
+  `,
+
+  css: `
+    ^ {
+      overflow: hidden;
+    }
+
+    ^ .search {
+      float: right;
+    }
+
+     ^ .menuOpen {
+      left: -00px;
+      transition: .2s;
+    }
+
+    ^ .menuClosed {
+      left: -52rem;
+      transition: .2s;
+    }
+
+    ^ .right .foam-u2-search-TextSearchView {
+      margin: 0;
+      height: 4rem;
+      position: relative;
+    }
+    ^ .right {
+      flex: 1;
+    }
+
+
+    ^browse-view-container {
+      overflow: scroll;
+    }
+
+    ^ .create-btn {
+      position: fixed;
+      bottom: 3rem;
+      right: 3rem;
+      border-radius: 100%;
+      padding: 3rem;
+      color: white;
+      font-weight: 900;
+      font-size: 4rem;
+      background-color: /*%PRIMARY2%*/ #937dff;
+      box-shadow: inset 0px 0px 15px 0px white;
+    }
+
+    ^ .back-btn {
+      background-color: blue;
+    padding: 1.5rem;
+    }
+
+    ^ .back-btn i {
+      font-size: 5rem;
+      float: left;
+      background-color: unset;
+      color: white;
+    }
+
+    .boxless-for-drag-drop {
+      background-color: unset;
+      border: unset;
+    }
+
+    .prof-icon {
+      height: 6rem;
+      border-radius: 50%;
+    }
+
+    .logout {
```

Comment:
Can you scope these with ^ so that if someone else creates classes with the same name they won't be affected? And/or scope the names when you generate them.